examples/hello-world-python/README.md

```diff
@@ -0,0 +1,247 @@
+# Hello World Python
+
+The is a complete "hello world" Python example to show you how to use Shroud.
+A development environment (container) is provided if needed, otherwise you
+should look at the [Dockerfile](Dockerfile) and ensure that you have basic d
+dependencies installed.
```

Comment:
This is a nice tutorial!

Side question: A Python `venv` could also work for this example/tutorial, right?

Reply:
I wouldn't expect you to look at everything too closely.
The Python tutorial was contributed by someone else at the Lab. The docker seemed like overkill but it's what they had. I agree that a virtual environment is simpler.
I'm going to go ahead and merge to get this out of the way then fix the typos after that.
Thanks for the review.  I'll try to release a bit more often in the future.